go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cilium/cilium-cli
 
-go 1.21
+go 1.21.0
```

Comment:
change added by `go mod tidy`

Reply:
This is fine. See also https://github.com/cilium/cilium/pull/27820. I think this change requires an update to the renovate configuration though.